package.json

```diff
@@ -16,17 +16,16 @@
     "tslint": "tslint -p tsconfig.json",
     "coverage": "node ./node_modules/nyc/bin/nyc.js --reporter=lcov --reporter=text-summary ./node_modules/tape/bin/tape ./test/index.js",
     "coveralls": "npm run coverage && node ./node_modules/coveralls/bin/coveralls.js <coverage/lcov.info",
-    "localchain": "./scripts/localchain.js",
-    "localchain:test": "NODE_ENV=testing LOCALCHAIN_PORT=8545 ./scripts/localchain.js",
-    "dev": "npm run build:ui && ./scripts/run.sh 80 repository db",
+    "localchain": "./src/localchain.js",
+    "localchain:test": "NODE_ENV=testing LOCALCHAIN_PORT=8545 ./src/localchain.js",
```

Comment:
Are you moving these to `src` because they're JS or...?

Reply:
Basically yes, but would probably be more correct to leave them in scripts. Will move that back there.